src/components/input/input.component.ts

```diff
@@ -247,6 +247,12 @@ export default class ZnInput extends ZincElement implements ZincFormControl {
 
   private handleBlur() {
     this.hasFocus = false;
+    if (this.hasAttribute('min') && this.value < this.min ) {
+      this.setAttribute('value', this.min as string );
+    }
+    if (this.hasAttribute('max') && this.value > this.max ) {
+      this.setAttribute('value', this.max as string );
+    }
```

Comment:
Instead of using `this.attribute`, you should just target the `@property value` instead. That way the reactive controller will pick up on the watch events of value.  I'm also not convinced this is correct